test/cursor.test.ts

```diff
@@ -464,6 +465,125 @@ databases.forEach(([kind, db]) => {
         expect(decodedCursor).toEqual({ id: "0" });
       });
     });
+
+    // MySql doesn't support NULLS LAST syntax
+    describe.skipIf(kind === "mysql")(
```

Comment:
This came as a surprise to me after I finished everything else. Wdyt around this? There's currently no check in the orderBy clause what dialect we're using. So you could potentially shoot yourself in the foot by specifying a `nullOrder` when also using mysql.

I posted a question [here](https://github.com/kysely-org/kysely/issues/714) to see if there's any native kysely support incoming. That would probably solve this since kysely would have to convert the `NULLS LAST` syntax to something `mysql` accepts.